feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-base.component.ts

```diff
@@ -22,6 +35,17 @@ export class ConfiguratorAttributeBaseComponent {
   private static PREFIX_DDLB_OPTION_PRICE_VALUE = 'option--price';
   protected static MAX_IMAGE_LABEL_CHARACTERS = 16;
 
+  rerender$: Observable<boolean> = of(true); // no delta rendering - always render directly only once
+  protected initDeltaRendering(
+    isDeltaRendering = false,
+    attributeKey?: string
+  ) {
+    if (isDeltaRendering && this.configuratorDeltaRenderingService) {
+      this.rerender$ =
+        this.configuratorDeltaRenderingService.rerender(attributeKey);
```

Comment:
[MEDIUM]

`rerender()` vs `getRerenderTrigger()`?
`rerender$` vs `rerenderTrigger$`?

IMHO the method name `rerender()` is a bit misleading - as it tells me the re-rendering will happen somehow within this method, as a side effect of it.
Instead, this method only returns an observable which should be a trigger for rerendering the consumer components.